_maps/map_files/RandomRuins/LavaRuins/lavaland_surface_althland_excavation.dmm

```diff
@@ -1145,12 +1145,9 @@
 /area/lavaland/surface/outdoors)
 "Es" = (
 /obj/structure/lattice/catwalk/mining,
-/obj/structure/ladder/unbreakable/dive_point{
-	desc = "The shattered remnants of a ladder hanging over the pitch-black chasm, providing no safe route to the abyss beneath. Its fractured rungs suggest an inevitable plunge, as if the remainder might soon join the void.";
-	name = "shattered ladder"
-	},
```

Comment:
I suggest you keep this desc and name for the ladder, and change the name of the path too